content/es/docs/home/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Documentación sobre Kubernetes
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage
+linkTitle: "Home"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "Documentación"
+    weight: 20
+    post: >
+      <p>Aprenda a usar Kubernetes con documentación conceptual, tutoriales y manuales de referencia. ¡Incluso puedes <a href="/editdocs/" data-auto-burger-exclude> contribuir con la documentación</a>! </p>
+overview: >
+  Kubernetes es orquestrador de contenedores de código abierto para automatizar la implementación, el escalado y la administración de aplicaciones en contenedores. El proyecto de código abierto está alojado por la Cloud Native Computing Foundation.(<a href="https://www.cncf.io/about">CNCF</a>).
```

Comment:
Kubernetes es _un orquestador_

Typo in "orquestrador" -> "orquestador". Also missing "un".

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/4c1b842159089b29b3d6a9f3c179d1393e4dec70